lang/el.po

```diff
@@ -0,0 +1,39 @@
+msgid ""
+msgstr ""
+"Language: el\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "msg_help_usage"
+msgstr "Χρήση: vrms-rpm [επιλογές]"
+msgid "msg_help_option_ascii"
+msgstr "Εμφανίζει rms ASCII-art όταν δεν βρεθούνε  nonfree πακέτα, \ n"
```

Comment:
`\n` is an escape code that translates to a line break. There must NOT be a space between the two characters.

Reply:
i know about this characters usage from Bash and Java.
But unfortunately my mobile text editor wasn't showing the spaces ... in my device there wasn't any space 😅  .. any way i fix theme from a PC . sorry for that i know now that i cant trust my mobile editor 😄 .